templates/components/panels/get-involved.hbs

```diff
@@ -4,20 +4,20 @@
       <h2>Get involved</h2>
       <div class="highlight"></div>
     </header>
-    <div class="row media">
-      <div class="six columns" id="read-rust">
+    <div class="flex flex-column flex-row-l">
+      <div id="read-rust" class="mw-50-l mh2-l pt0 flex flex-column justify-between-l">
         <h3>Read Rust</h3>
         <p>We love documentation! Take a look at the books available online, as well as key blog posts and user guides.</p>
         <a href="learn" class="button button-secondary">Read the book</a>
       </div>
-      <div class="six columns" id="watch-rust">
+      <div id="watch-rust" class="mw-50-l mh2-l pt0 flex flex-column justify-between-l">
         <h3>Watch Rust</h3>
         <p>The Rust community has a dedicated YouTube channel collecting a huge range of presentations and
         tutorials.</p>
         <a href="https://www.youtube.com/channel/UCaYhcUwRBNscFNUKTjgPFiA" class="button button-secondary">Watch the Videos</a>
       </div>
     </div>
-    <div class="pt3">
+    <div>
```

Comment:
In the current website there is a divider between these two rows, but with the PR there is basically no white space between them. Can you add some margin?

The margin should be added also between the other two columns on mobile.

Reply:
Good catch! Added the spacing back in.